src/tribler-core/tribler_core/components/bandwidth_accounting/bandwidth_accounting_component.py

```diff
@@ -14,10 +17,9 @@
 from tribler_core.restapi.rest_manager import RESTManager
 
 
-class BandwidthAccountingComponent(Component):
+class BandwidthAccountingComponent(RestfulComponent):
     community: BandwidthAccountingCommunity
 
-    _rest_manager: RESTManager
     _ipv8: IPv8
 
     async def run(self):
```

Comment:
If you add a class to the chain of inheritance, it is almost always better to call overridden methods of a base class, eg:

```python
class BandwidthAccountingComponent(RestfulComponent):
    community: BandwidthAccountingCommunity

    _ipv8: IPv8

    async def run(self):
        await super().run()
```

Overwise you change the behavior of base class kinda in an "illegal" way.

Reply:
Agree, changed it to calling the base method